File: BookHotel/BookHotel/Data/Entities/Hotel.cs
using BookHotel.Auth.Model;
using System.ComponentModel.DataAnnotations;

namespace BookHotel.Data.Entities
{
	public class Hotel : IUserOwnedResource
	{
		public int Id { get; set; }
		public string Name { get; set; }
		public string Address { get; set; }
		public string Email { get; set; }
		public int StarCount { get; set; }
		public int CityId { get; set; }
		public City City { get; set; }
		[Required]
		public string UserId { get; set; }
		public BookHotelRestUser User { get; set; }
	}
}

File: bookhotel-frontend/src/cities/models/Hotel.ts

export default class Hotel {
  id: string;
  name: string;
  address: string;
  email: string
  starCount: string;

  constructor(dto: any) {
    this.id = dto._id;
    this.name = dto.name;
    this.address = dto.address;
	this.email = dto.email;
	this.starCount = dto.starCount;
  }
}
